var/spack/repos/builtin/packages/charmpp/package.py

```diff
@@ -237,7 +241,8 @@ def install(self, spec, prefix):
         # here.
         options = [
             os.path.basename(self.compiler.cc),
-            os.path.basename(self.compiler.fc),
+            'gfortran'
+            if '@:6.8.2 %aocc' in spec else os.path.basename(self.compiler.fc),
```

Comment:
I don't think we should hard-code `gfortran` here. What is the intent of this change? Maybe we can reach the same result in a more idiomatic way.

Reply:
charm has limitation for` flang` as fortran compiler for versions 6.8.2 and below, so added `gfortran` as cross compilation option.

Reply:
We cannot be ensured that `gfortran` is there. Is there an option to turn off Fortran? Do you need Fortran support for NAMD?

Reply:
We don't need fortran support NAMD

Reply:
we don't need `fortan` support for NAMD

Reply:
Hi @alalazo, I have addressed the above issue please review it.